java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -41,6 +42,9 @@ public void transaction_builds_protobuf_request() {
                                 .addArgs(RETURN_OLD_VALUE)
                                 .build()));
 
+        transaction.del(new String[] {"key1", "key2"});
```

Comment:
Why not add these tests to test utilities instead?

Reply:
That's for IT tests. This is for UT tests.  
That being said... we could probably create a BaseTransactionTests.java file that does something similar.